### ISSUE TITLE
Annotated moves showing more than once

### ISSUE THREAD

**Issue by josec87** [marcusbuffett/chessmadra-frontend#16]
Annotated moves are repeated.
![image](https://user-images.githubusercontent.com/747418/206814857-e6f63eba-b6d6-460b-adb2-fee112d5d377.png)

**Comment by alexandru-duca** [comment]
I wasn't able to reproduce this bug.

I started a new repertoire for black from scratch and added the line `1. e4 c6 2. d4 d5 3. Nc3 dxe4 4. Nxe4 Nf6`, which reaches the same position as shown in @josec87's picture.

![Screenshot from 2023-02-09 17-18-50](https://user-images.githubusercontent.com/107179714/217871968-85106b84-fe28-4dfe-8c68-045458ad7c75.png)

Here, the annotated move is not repeated. Editing the annotation doesn't change anything. I tried out all the different settings (Rating System, Rating Range, Position Frequency) and the annotated move was never displayed twice.
